packages/bits/src/lib/wizard-v2/wizard.directive.ts

```diff
@@ -1,5 +1,6 @@
 import {
```

Comment:
@DmytroPutrin, do you mind adding some unit tests for the new functionality?

Reply:
in case we have no spec files for wizard 2, here separate task for ut (NUI-5070).